manager/api/rest-impl/src/main/java/io/apiman/manager/api/rest/impl/OrganizationResourceImpl.java

```diff
@@ -1697,6 +1698,9 @@ protected ApiVersionBean createApiVersionInternal(NewApiVersionBean bean,
         if (bean.getParsePayload() != null) {
             newVersion.setParsePayload(bean.getParsePayload());
         }
+        if (bean.getDisableKeysStrip() != null) {
```

Comment:
Does this also work if you clone an API?

Reply:
should be, but I realized that when I try to create a clone of existing api, some settings are not copied (payload inspection ...). I'll take care of it later